src/effects/How to make an effect stateless.txt

```diff
@@ -0,0 +1,189 @@
+4 STEPS TO MAKE AN EFFECT STATELESS
+
+1) "Define EffectEchoSettings"
+
+   In header file:
+       before the effect definition, define a struct EffectEchoSettings containing the settings.
+
+       Base class remains: StatefulPerTrackEffect
```

Comment:
We may transform other effects that are not per-track effects but will be made stateless too